state/modelsummaries.go

```diff
@@ -71,6 +71,7 @@ type ModelSummary struct {
 
 	// Needs Migration collection
 	// Do we need all the Migration fields?
+	// Migration needs to be a pointer as we may not always have one.
```

Comment:
ModelMigration is already an 'interface" which means it can already be set to 'nil'. Otherwise I would agree.

Reply:
Yes, I think this comment was me thinking out loud :) Please remove it :)